pkg/services/sqlstore/migrations/ualert/ualert.go

```diff
@@ -243,10 +243,27 @@ func (m *migration) Exec(sess *xorm.Session, mg *migrator.Migrator) error {
 		return err
 	}
 
+	if err := m.writeAlertmanagerConfig(&amConfig, allChannels); err != nil {
+		return err
+	}
+
+	if err := m.writeSilencesFile(); err != nil {
+		m.mg.Logger.Error("alert migration error: failed to write silence file", "err", err)
+	}
+
+	return nil
+}
+
+func (m *migration) writeAlertmanagerConfig(amConfig *PostableUserConfig, allChannels map[interface{}]*notificationChannel) error {
+	if len(allChannels) == 0 {
+		// No channels, hence don't require Alertmanager config.
```

Comment:
I don't think it's worth another CI run but you address it in any other of the open PRs - can we log here stating there's nothing to do or log that we have X channels to migrate.

Reply:
I will just do it in this PR :P else its easy to forget

Reply:
I will log for nothing to do, number of channels wont make sense because they can be duplicated across receivers.